### PR TITLE
refactor: apply identity v2 base

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

- site-initialize process
- sign-in process
- identity reference

### CODE REVIEW

File: apps/web/src/store/modules/workspace/workspace-store.ts [hunk 1]
```diff
@@ -26,17 +24,16 @@ export const useWorkspaceStore = defineStore('workspace-store', () => {
...
     });
 
     const actions = {
-        async load() {
+        async load(domainId) {
             try {
-                const { results } = await SpaceConnector.clientV2.inventory.workspace.list({});
+                const { results } = await SpaceConnector.clientV2.identity.workspace.list({
+                    domain_id: domainId,
+                });
```
Comment (yuda110) [inline apps/web/src/store/modules/workspace/workspace-store.ts:36]: nitpick: How about add 'TODO' for when `domain_id` becomes unnecessary?

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: packages/core-lib/src/space-connector/token-api.ts [hunk 2]
```diff
@@ -100,7 +100,7 @@ export default class TokenAPI {
         if (TokenAPI.checkRefreshingState() !== 'true') {
             try {
                 TokenAPI.setRefreshingState();
-                const response: AxiosPostResponse = await this.refreshInstance.post(REFRESH_URL);
+                const response: AxiosPostResponse = await this.refreshInstance.post(REFRESH_URL, {});
```
Comment (sulmoJ) [inline packages/core-lib/src/space-connector/token-api.ts:103]: question: Why added second parameter(`{}`)??

File: apps/web/src/store/modules/user/actions.ts [hunk 3]
```diff
@@ -73,16 +70,18 @@ const getUserInfoFromToken = (token: string): string[] => {
...
 
-const getUserRoleBindings = async (userId: string): Promise<Array<UserRole>> => {
+const getUserRoleBindings = async (domainId): Promise<Array<UserRole>> => {
     try {
         const userRoles: Record<string, UserRole> = {};
-        const { results } = await SpaceConnector.client.identity.roleBinding.list({
-            resource_type: 'identity.User',
-            resource_id: userId,
+        const { results } = await SpaceConnector.clientV2.identity.role.list({
+            domain_id: domainId,
```
Comment (sulmoJ) [inline apps/web/src/store/modules/user/actions.ts:77]: nitpick: TODO comment should be added here too!

File: apps/web/src/store/modules/user/actions.ts [hunk 4]
```diff
@@ -5,33 +5,39 @@ import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
...
+//         timezone: response.timezone,
+//         // email_verified : There is data only when the value is true.
+//         emailVerified: !!response.email_verified,
+//     };
+// };
+
+const getUserInfo = async (userId: string, domainId: string): Promise<Partial<UserState>> => {
+    const response = await SpaceConnector.clientV2.identity.user.get({
+        user_id: userId,
+        domain_id: domainId,
```
Comment (sulmoJ) [inline apps/web/src/store/modules/user/actions.ts:32]: nitpick: TODO comment should be added here too!

File: apps/web/src/store/modules/user/actions.ts [hunk 5]
```diff
@@ -102,27 +101,25 @@ const getUserRoleBindings = async (userId: string): Promise<Array<UserRole>> =>
 };
 
 export const signIn = async ({ commit }, signInRequest: SignInRequest): Promise<void> => {
-    const response = await SpaceConnector.client.identity.token.issue({
-        domain_id: signInRequest.domainId,
-        user_id: signInRequest.userId || null, // user_id is nullable
-        user_type: signInRequest.userType,
+    const domainId = signInRequest.domainId;
+    const response = await SpaceConnector.clientV2.identity.token.issue({
+        domain_id: domainId,
```
Comment (sulmoJ) [inline apps/web/src/store/modules/user/actions.ts:106]: nitpick: TODO comment should be added here too!

**Review by sulmoJ — APPROVED** [review]
New v2 apis are so beautiful!!
Thank you for your hard work. 
I left some comments~~

**Pull request merged**